1-exercises/D-do-while/exercise.js

```diff
@@ -7,9 +7,18 @@
 */
 
 function evenNumbersSum(n) {
-    // TODO
+  // TODO
+  let i = 0;
+  let j = 0;
+  let sum = 0;
+  do {
+    sum += j;
+    j += 2;
+    i++;
+  } while (i < n);
+  return sum;
```

Comment:
Again, more descriptive variable names would make it more easier to read, but technically it looks great!